examples/config-sign.json

```diff
@@ -0,0 +1,57 @@
+{
+	"version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/home/ijigau/Downloads/zot-repo"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080",
+        "ReadOnly": false
+    },
+    "log": {
+        "level": "debug"
+    },
+	"extensions": {
+		"sign": {
+            "enable": false,
+            "verificationInterval": "1m",
+            "publicKeys":{
+                "alpine": "/home/ijigau/Downloads/pass1/cosign.pub",
+                "repo": "keypath"
+            },
+            "publicKeys3": [
```

Comment:
Make this a per-repo config (invert the structure)
"repositories": [{
"prefix":<>,
"publicKey": <>,
},
{
...
}
]